extensions/amp-analytics/0.1/requests.js

```diff
@@ -116,13 +117,24 @@ export class RequestHandler {
     const triggerParams = trigger['extraUrlParams'];
     const isImmediate =
         (trigger['immediate'] === true) || (this.maxDelay_ == 0);
+
+    const isV2ExpansionON = isExperimentOn(this.ampdoc.win,
+        'url-replacement-v2');
+    let bindings;
+    if (isV2ExpansionON) {
+      const macros = this.variableService_.getMacros();
+      bindings = Object.assign({}, dynamicBindings, macros);
```

Comment:
you may want to move `dynamicBindings` to variable as well in future PRs : )

Reply:
moved this around when flag position changed